### ISSUE TITLE
Disable hook if commit message already starts with emoji

### ISSUE THREAD

**Issue by thdk** [carloscuesta/gitmoji-cli#1037]
### Description of the problem

Context: automating bumping versions of node packages

```sh
❯ gitmoji -i
❯ git commit -a -m "🔖 Bump version to $npm_package_version"
? Choose a gitmoji: 🔖  - Release / Version tags.
? Enter the commit title [18/48]: 🔖 Bump version to 0.1.0
? Enter the commit message: 
[feature/abc 468583a] 🔖 🔖 Bump version to 0.1.0
 2 files changed, 4 insertions(+), 4 deletions(-)
```

Could we detect in the prepare commit message hook that the commit message already contains an emoji and if so skip the emoji prompt?

Or still show the prompt but preselect the result with the emoji from the commit message?

### Solution

```
const commit = (options: CommitOptions): Function => {
  if (options.mode === COMMIT_MODES.HOOK) {
  registerHookInterruptionHandler()
    if (options.message?.startsWith(anyEmoji)) {
        commit(options)
     else {
        
        return cancelIfNeeded().then(() => promptAndCommit(options))
    }
  }

  return promptAndCommit(options)
}
```

### Alternatives

`git commit --no-verify` doesn't seem to prevent the gitmoji hook.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/carloscuesta/gitmoji-cli/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/carloscuesta/gitmoji-cli/blob/master/.github/CONTRIBUTING.md).
- [X] Check that there isn't already an issue requesting the same feature.

**Comment by carloscuesta** [comment]
Hey!

Thanks for opening the issue, I think it makes sense but we should probably aim to support the `--no-verify` option instead of doing this "magic" of skipping the hook if the emoji is provided already 

Feel free to raise a Pull Request! 🙏🏼

**Comment by carloscuesta** [comment]
Hey! So I took at stab at this and found out it's not possible to use `--no-verify` for the `prepare-commit-msg` hook (it is ignored) [as mentioned in the docs](https://git-scm.com/docs/githooks#_prepare_commit_msg)

<img width="681" alt="CleanShot 2023-05-31 at 17 44 37@2x" src="https://github.com/carloscuesta/gitmoji-cli/assets/7629661/59a8078f-5772-4df9-b16c-d927e015abae">

I implemented the PR as an alternative solution to this.

We might want to explore switching from `prepare-commit-msg` to `commit-msg` to support `--no-verify`

**Comment by carloscuesta** [comment]
Released in `v8.3.0` 🔖 https://github.com/carloscuesta/gitmoji-cli/releases/tag/v8.3.0
